### PR TITLE
D3 prototype with project.csv data and animated resorting

### PR DESCRIPTION
**Pull request by ostermanj**
This is much closer to the block visualizations from the prototype team. Might as well have it on live site where we can point people to it.

### CODE REVIEW

**Comment by NealHumphrey** [comment]
@ostermanj this looks awesome! Definitely coming along. I made a few small comments - 2 quick notes on comments, and one addition of a link between index.md and early-examples. Should take you 2 minutes, I'll let you make the changes rather than doing it myself, though. Other comments are just for future reference regarding naming conventions and design patterns.

**Comment by ostermanj** [comment]
@NealHumphrey Thanks! Where should I look for your comments?

**Comment by NealHumphrey** [comment]
@ostermanj if you view this pull request on Github, they show up in the comment stream above. You can also see them in context by clicking the 'files changed' tab at the top of the pull request. 
Direct link: https://github.com/codefordc/housing-insights/pull/62

File: docs/d3-mockup/js/prototype-1a.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+            var chart = this;
+            chart.svg.selectAll('rect')
+            
+            .transition().delay(250).duration(750)
+            .attr('fill-opacity', function(d) { // opacity is a function of the value
+                    return chart.scale(d[field]); // takes the field value and maps it according to the scaling function defined above
+                }) 
+        },
+
+        resort: function(value){
```
Comment (ostermanj) [inline docs/d3-mockup/js/prototype-1a.js]: agreed. added comment to that effect

File: docs/d3-mockup/index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+**Initial sort is by zip code (top to bottom then right)**
+
+a.
+<div id="chart-0" class="d3-chart"></div>
+<!-- next prototype goes below -->
+<!--b.
+<div id="chart-1"></div>-->
+Codebase: [https://github.com/codefordc/housing-insights/tree/dev/docs/d3-mockup](https://github.com/codefordc/housing-insights/tree/dev/docs/d3-mockup)
+
```
Comment (ostermanj) [inline docs/d3-mockup/index.md]: done

**Comment by ostermanj** [comment]
Pushed a commit in response to your comments, @NealHumphrey. Thanks!

**Pull request merged**